mfxconfig.h.in

```diff
@@ -34,3 +34,7 @@
 #if defined(MFX_ENABLE_VP8_VIDEO_DECODE) && defined(MFX_VA) &&!defined(AS_VP8D_PLUGIN)
 #define MFX_ENABLE_VP8_VIDEO_DECODE_HW
 #endif
+
+#if !defined(AS_HEVCD_PLUGIN)
```

Comment:
I do not think that AS_HEVCD_PLUGIN is actually needed any more at the first place. Let's try to dispose of them as a first step. I think that the only one which may potentially be needed is AS_HEVCE_PLUGIN and maybe same for LA.